options.go

```diff
@@ -154,3 +155,14 @@ func WithoutHMAC() Option {
 		})
 	}
 }
+
+func ParsePaths(fields ...*string) error {
+	for i := 0; i < len(fields); i++ {
+		if newVal, err := parseutil.ParsePath(*fields[i]); err != nil {
```

Comment:
Shouldn't this trap `errors.Is(err, ErrNotAUrl)` and not return it leaving whatever the field value was previously?

I'm just wondering if we considered with these little tidbits but all three are me trying to poke holes into and I don't feel strongly about them.

1.  If we were given any values that started or ended with spaces they will no longer work as ParsePath will trim them.
2. It's a little unfortunate that with `env://` values we don't get an error back saying it didn't exist which may be hard for operators to diagnose connection issues in the future, but we do for file based schemas
3. Obviously if the values being passed in had values beginning with file:// or env:// that were for some reason meant to be passed to the underlying wrapper there is no way out now.